script/compiler_base.yml

```diff
@@ -61,9 +61,8 @@
     GIT_SUBMODULE_STRATEGY: normal
     CUPLA_CXX: "hipcc"
     ALPAKA_ACCS: "alpaka_ACC_GPU_HIP_ENABLE"
-    # architecture of the Vega 64
-    CUPLA_CMAKE_ARGS: "-DGPU_TARGETS=gfx900"
   before_script:
+    - source scripts/install_hip.sh
```

Comment:
I suggest to use `${CI_PROJECT_DIR}/scripts/install_hip.sh`.

Reply:
`${CI_PROJECT_DIR}/scripts/install_hip.sh` -> `${CI_PROJECT_DIR}/script/install_hip.sh`. Remove the s from scripts